src/Native/System.Native/pal_runtimeinformation.h

```diff
@@ -14,5 +14,6 @@ enum
 {
     ARCH_X86,
     ARCH_X64,
-    ARCH_ARM
+    ARCH_ARM,
+    ARCH_ARM64
```

Comment:
The managed counterpart to this:
https://github.com/dotnet/corefx/blob/master/src/Common/src/Interop/Unix/System.Native/Interop.GetUnixArchitecture.cs#L15
needs to be kept in sync.

And until the public surface area is updated (which will need an API review as I previously mentioned), this function:
https://github.com/dotnet/corefx/blob/master/src/System.Runtime.InteropServices.RuntimeInformation/src/System/Runtime/InteropServices/RuntimeInformation/RuntimeInformation.Unix.cs#L40
will end up failing with an assert/exception if it's invoked on the new platform.  @Priya91, what do you want to do here on the managed side?  Map ARM64 to ARM?  Throw a more explicit exception if the result comes back as something unknown (to handle this and future expansions)?  Something else?